simulation/generateSearchQueries.js

```diff
@@ -5,22 +5,22 @@ const BATCH_SIZE = 1000;
 const SEARCH_QUERY_ID_START = 5000000;
 const VISIT_ID_START = 1;
 const USER_ID_RANGE = 100000;
-const SEARCH_FREQUENCY_IN_SECONDS = 30;
+const SEARCH_FREQUENCY_IN_MS = 100;
 const MARKETS = [
   'San Francisco', 'Seattle', 'Sydney', 'New York', 'Toronto', 'Paris',
   'London', 'Hong Kong', 'Amsterdam', 'Montreal',
 ];
 const MAX_DAYS_UNTIL_TRAVEL = 45;
 const MAX_LENGTH_OF_STAY = 7;
-const ROOM_TYPES = ['any', 'entire home', 'private', 'shared room'];
+const ROOM_TYPES = ['any', 'Entire home/apt', 'Private room', 'Shared room'];
 
 /* ----- HELPER FUNCTIONS BEGIN ----- */
 
-const incrementIdFrom = (num) => {
-  let startId = num;
+const incrementIdFrom = (startId) => {
+  let id = startId;
```

Comment:
could this be better written as such? : 
const incrementIdFrom = (id) => {
  return id += 1
}